include/GafferImage/Format.h

```diff
@@ -132,6 +133,10 @@ GAFFERIMAGE_API std::ostream & operator << ( std::ostream &os, const GafferImage
 
 } // namespace GafferImage
 
+namespace IECore {
+inline void murmurHashAppend( IECore::MurmurHash &h, const GafferImage::Format &data );
+}
```

Comment:
This should just be in the GafferImage namespace - argument dependent lookup means it just works.

Reply:
Fixed in c75bcea